sormas-ui/src/main/java/de/symeda/sormas/ui/caze/BulkCaseDataForm.java

```diff
@@ -209,12 +236,15 @@ protected void addFields() {
 		ComboBox facility = addInfrastructureField(CaseBulkEditData.HEALTH_FACILITY);
 		facility.setImmediate(true);
 		facility.setEnabled(false);
-
 		region.addValueChangeListener(e -> {
 			RegionReferenceDto regionDto = (RegionReferenceDto) e.getProperty().getValue();
 			FieldHelper
 				.updateItems(district, regionDto != null ? FacadeProvider.getDistrictFacade().getAllActiveByRegion(regionDto.getUuid()) : null);
 		});
+		healthFacilityDetails.addValueChangeListener(e -> {
+			this.getValue().setHealthFacilityDetails(healthFacilityDetails.getValue());
```

Comment:
Why is this necessary? The healthFacilityDetails entered should automatically be added to the bound CaseBulkEditData.